ui/src/main/java/com/adyen/checkout/ui/theme/AdyenButtonStyles.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2025 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by oscars on 10/4/2025.
+ */
+
+package com.adyen.checkout.ui.theme
+
+data class AdyenButtonStyles(
+    val cornerRadius: Int? = null,
```

Comment:
Do you think we can use `Dp` here?

Reply:
`Dp` is a compose class and the aim is to keep our system agnostic. I was thinking to make clear with kdocs and/or annotations that this value is in dp